docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+filter plugin can have a significant effect on performance. The grok filter’s regular expression computation is 
+particularly resource-intensive. One way to better address this increased demand for computing resources is to use 
+parallel processing on multicore machines. Use the -w switch to set the number of execution threads for Logstash 
+filtering tasks. For example the bin/logstash -w 8 command uses eight different threads for filter processing.
+
+image::images/deploy_2.png[]
+
+// [float]
+[[deploying-logstash-forwarder]]
+=== Using Logstash Forwarder
+
+The https://github.com/elastic/logstash-forwarder[Logstash Forwarder] is a lightweight, resource-friendly tool written 
+in Go that collects logs from files on the server for processing on other machines. The Logstash Forwarder acts as a 
+client to a centralized Logstash instance, communicating using a secure protocol called Lumberjack. The Logstash 
+instance that acts as a server must be configured to use the {logstash}plugins-inputs-lumberjack.html[Lumberjack 
+input plugin].
+
+The Logstash Forwarder uses the computing resources of the machine hosting the source data, and the Lumberjack input 
+plugin minimizes the resource demands on the Logstash instance, making this architecture attractive for use cases with 
+resource constraints.
+
+image::images/deploy_3.png[]
+
+// [float]
+[[deploying-larger-cluster]]
+=== Scaling to a Larger Elasticsearch Cluster
+
+Typically, Logstash does not communicate with a single Elasticsearch node, but with a cluster made up of several nodes. 
+Logstash can use any of the protocols Elasticsearch makes available to move data into the cluster: 
+{guide}_transport_client_versus_node_client.html[HTTP, transport, or node]. 
+
+You can use the HTTP REST APIs that Elasticsearch provides to index data into the Elasticsearch cluster, transporting 
+the data in a JSON representation. Using the REST APIs does not require the Java client classes or any additional JAR 
+files and has no performance disadvantages compared to the transport or node protocols. You can secure communications 
+that use the HTTP REST APIs with the {shield}[Shield] plugin, which supports SSL and HTTP basic authentication.
+
+You can also use the Java APIs that Elasticsearch provides to serialize the data into a binary representation, using 
+the transport protocol. The transport protocol has the option to sniff the endpoint of the request, selecting an 
+arbitrary client or data node in the Elasticsearch cluster. 
+
+Both the HTTP and transport protocols separate Logstash from the Elasticsearch cluster. The node protocol, by contrast, 
+has the machine running the Logstash instance join the Elasticsearch cluster, running an Elasticsearch instance. The 
+data that needs indexing propagates from this node to the rest of the cluster. Since the machine is part of the 
+cluster, the cluster topology is available, making the node protocol a good fit for use cases that use a relatively 
```

Comment:
@debadair a node protocol joins a cluster and gets the cluster state, so it knows where the shards and machines are located. To this end node saves a hop to index data (LS->data node).. otherwise you have to LS->ES node->Data node